src/com/maddyhome/idea/vim/action/motion/mark/MotionGotoFileMarkAction.java

```diff
@@ -34,8 +36,19 @@ public MotionGotoFileMarkAction() {
   }
 
   private static class Handler extends MotionEditorActionHandler {
-    public int getOffset(@NotNull Editor editor, DataContext context, int count, int rawCount, @NotNull Argument argument) {
-      return VimPlugin.getMotion().moveCaretToFileMark(editor, argument.getCharacter());
+    Handler() {
+      super(true);
+    }
+
+    @Override
+    public int getOffset(@NotNull Editor editor, @NotNull Caret caret, @NotNull DataContext context, int count,
+                         int rawCount, @Nullable Argument argument) {
+      if (argument == null) {
+        return -1;
+      }
+
+      final char mark = argument.getCharacter();
+      return VimPlugin.getMotion().moveCaretToMark(editor, caret, mark, false);
```

Comment:
It seems that jumping to a mark from another file has stopped working. Please check both file mark jumps and file line mark jumps.